src/components/calculator/PrevalenceControls.tsx

```diff
@@ -139,6 +142,11 @@ export const PrevalenceControls: React.FunctionComponent<{
   setter: (newData: CalculatorData) => void
 }> = ({ data, setter }): React.ReactElement => {
   const { t } = useTranslation()
+  for (const iso_code of Object.keys(i18n.services.resourceStore.data)) {
+    countries.registerLocale(
+      require('i18n-iso-countries/langs/' + iso_code + '.json'), // eslint-disable-line @typescript-eslint/no-var-requires
```

Comment:
Curious for @beshaya's thought son this coding pattern.

Reply:
This is what https://github.com/michaelwittig/node-i18n-iso-countries suggests, as it does not load all countries by default in browser contexts to save bandwidth. If it needs to be changed, I'd need some help, I'm beyond my depth in React/TypeScript here.

Reply:
Okay. I'm good with leaving it as-is then! Let's call this point good.